src/main/app/claims/models/claim.ts

```diff
@@ -32,14 +33,14 @@ export class Claim {
   countyCourtJudgmentRequestedAt: Moment
   countyCourtJudgmentIssuedAt: Moment
   response: Response
+  claimantResponse: ClaimantResponse
   defendantEmail: string
   settlement: Settlement
   settlementReachedAt: Moment
   totalAmountTillToday: number
   totalAmountTillDateOfIssue: number
   totalInterest: number
   features: string[]
-  directionsQuestionnaireDeadline: Moment
```

Comment:
Are you sure this needs to be removed? If we don't need this now, please remove it from `claim-store-api` as well